queue/queue.py

```diff
@@ -3,13 +3,14 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = []
 
   def enqueue(self, item):
-    pass
+    self.storage.insert(0,item)
   
-  def dequeue(self):
-    pass
+  def dequeue(self): 
+    if self.storage != []:
+      return self.storage.pop()
 
   def len(self):
-    pass
+    return len(self.storage)
```

Comment:
your tests are passing here because you are returning the length of self.storage. What you should be doing here is returning self. size and your tests would fail because you forgot to increment and decrement the size of queue as you added/removed items.